adapter/service/virtualmeter/manager.go

```diff
@@ -449,7 +432,7 @@ func (m *manager) createVirtualServicesForThing(t adapter.Thing) (outLvlSwitchSe
 					numericmeter.MeterElec,
 					m.ad.Name(),
 					m.ad.Address(),
-					t.Address(),
+					t.Address()+group,
 					t.InclusionReport().Groups,
```

Comment:
This is also wrong, you should not copy all groups, just a single group with which you're associating virtual services.

Reply:
Possibly just s.Specification().Groups would be enough.